src/commands/addBinding/settingSteps/eventHub/EventHubAuthRuleListStep.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { EventHubManagementClient, EventHubManagementModels } from 'azure-arm-eventhub';
+import { AzureWizardPromptStep, createAzureClient } from 'vscode-azureextensionui';
+import { localize } from '../../../../localize';
+import { IBaseResourceWithName, promptForResource } from '../../../../utils/azure';
+import { nonNullProp } from '../../../../utils/nonNull';
+import { IEventHubWizardContext } from './IEventHubWizardContext';
+
+export class EventHubAuthRuleListStep extends AzureWizardPromptStep<IEventHubWizardContext> {
+    public async prompt(wizardContext: IEventHubWizardContext): Promise<void> {
+        const namespaceName: string = nonNullProp(wizardContext, 'namespaceName');
+        const resourceGroupName: string = nonNullProp(wizardContext, 'resourceGroupName');
+        const eventHubName: string = nonNullProp(wizardContext, 'eventHubName');
+
+        const client: EventHubManagementClient = createAzureClient(wizardContext, EventHubManagementClient);
+
+        const namespaceDescription: string = localize('namespacePolicy', '(namespace policy)');
+        const hubDescription: string = localize('hubPolicy', '(hub policy)');
```

Comment:
We haven't been using `()` for any of the other quickpick descriptions.  Is this necessary?

Reply:
I added them because that's what the portal does:
![Screen Shot 2019-04-24 at 2 29 04 PM](https://user-images.githubusercontent.com/11282622/56695216-58f72000-669d-11e9-9313-93fa05511842.png)

Honestly I'm open to the discussion, but it's not unique to this PR so I'll leave it as-is for now:
![Screen Shot 2019-04-24 at 2 25 40 PM](https://user-images.githubusercontent.com/11282622/56695154-3d8c1500-669d-11e9-8879-3daab3912884.png)
![Screen Shot 2019-04-24 at 2 26 37 PM](https://user-images.githubusercontent.com/11282622/56695155-3d8c1500-669d-11e9-975a-f032d0a6e628.png)
![Screen Shot 2019-04-24 at 2 28 12 PM](https://user-images.githubusercontent.com/11282622/56695156-3d8c1500-669d-11e9-8439-7210aaaa80e8.png)